src/main/java/frc/robot/util/DriverJoystick.java

```diff
@@ -6,14 +6,48 @@
  * This class is wrapper class to allow the abstraction of buttons and joysticks (removes the need to specify ports every time you get a button)
  */
 public class DriverJoystick {
-    private Joystick driverJoy = new Joystick(Context.joystickID);
+    private Joystick joystick;
+
+    public DriverJoystick() {
+        joystick = new Joystick(Context.joystickID);
+    }
 
     public double getThrottle() {
-        return driverJoy.getRawAxis(Context.throttleAxisID);
+        return joystick.getRawAxis(Context.throttleAxisID);
     }
 
     public double getYaw() {
-        return driverJoy.getRawAxis(Context.yawAxisID);
+        return joystick.getRawAxis(Context.yawAxisID);
+    }
+
+    public double getRawAxis(int axis)
```

Comment:
The idea with the joystick wrapper is that we can create getter functions like getThrottle() or getRunAutoButton() so that code looks cleaner and there is no need for a getRawAxis() function. Putting this here defeats the purpose of a Wrapper class.